### PR TITLE
Adding netcore outputs to cred provider

### PR DESCRIPTION
**Pull request by shubham90**

### CODE REVIEW

File: build/build.yml [hunk 1]
```diff
@@ -101,6 +94,32 @@ steps:
     targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\'
     flattenFolders: true
 
+- task: ArchiveFiles@2
+  displayName: Create netcore tarball
+  inputs:
+    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
```
Comment (zarenner) [inline build/build.yml]: Not part of your change, but a bit confusing IMO that the 'tarball' folder is used to produce both a tarball/tgz, and a zip.

File: build/build.yml [hunk 2]
```diff
@@ -101,6 +94,32 @@ steps:
...
+    includeRootFolder: false
+    archiveType: 'zip'
+    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NetCore.NuGet.CredentialProvider.zip'
+    replaceExistingArchive: true
+
+- task: CopyFiles@2
+  displayName: Copy netfx files into tarball
+  inputs:
+    sourceFolder: '$(Build.SourcesDirectory)\CredentialProvider.Microsoft\bin\$(BuildConfiguration)\net461\'
```
Comment (zarenner) [inline build/build.yml]: net461 hardcoded? I assume it'll at least break the build if we upgraded netfx ver?

File: build/build.yml (hunk 1 above)
Comment (shubham90) [reply build/build.yml]: yeah, I saw that too, forgot to make the change with this PR though. Will change in a separate PR.

File: build/build.yml (hunk 2 above)
Comment (shubham90) [reply build/build.yml]: good catch! will fix this as weel in follow up PR.

**Pull request merged**